pom.xml

```
@@ -393,6 +393,34 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-release-plugin</artifactId>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M3</version>
+                <executions>
+                    <execution>
+                        <id>enforce-maven</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <bannedPlugins>
+                                    <level>ERROR</level>
+                                </bannedPlugins>
+                                <requireMavenVersion>
+                                    <version>[3.6,)</version>
+                                    <message>Invalid Maven version, the minimum required is 3.6.x.</message>
```

Enforce minimum version to be the same as what's on Jenkins.

```suggestion
                                    <version>[3.6.3,)</version>
                                    <message>Invalid Maven version, the minimum required is 3.6.3.</message>
```